draft-ietf-moq-transport.md

```diff
@@ -1806,6 +1771,61 @@ STREAM_HEADER_PEEP {
 ~~~
 
 
+## Fetch
+
+Subscribers issues fetch requests to prior objects while the subscriptions are made to the live edge. Fetch allows objects to be delivered at a requested rate. The fetched objects are delivered on the same stream as the request.
```

Comment:
Subscribers issues -> Subscribers issue
requests to prior objects -> requests for prior objects